services/user.py

```diff
@@ -0,0 +1,60 @@
+from django.db import models
+from django.core.exceptions import ValidationError
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.validators import validate_email
+
+from db.models import User
+
+
+def email_validator(email: str) -> None:
+    try:
+        validate_email(email)
+    except ValidationError:
+        raise ValidationError("Wrong email structure")
+
+
+def create_user(username: str,
+                password: str,
+                email: models.EmailField = None,
+                first_name: str = None,
+                last_name: str = None) -> None:
+    if email:
+        email_validator(email)
+
+    User.objects.create_user(username=username,
+                             email=(email or ""),
+                             first_name=(first_name or ""),
+                             last_name=(last_name or ""),
+                             password=password)
+
+
+def get_user(user_id: int) -> User | None:
+    try:
+        man = User.objects.get(id=user_id)
+    except ObjectDoesNotExist:
+        return None
+    return man
+
+
+def update_user(user_id: int,
+                username: str = None,
+                password: str = None,
+                email: models.EmailField = None,
```

Comment:
Email validators are usually used when working with form data. You don't really have to include it here but if you do, why not include validation in other fields then?

Reply:
I will rephrase myself. I don't think validation is necessary here - it usually is done on the model level. The same as you have done in your clean method on Ticket. 
And on the other hand I was wondering why did you choose to add extra validation only for email, but not to other fields and other models

Reply:
you don't have to do any extra validation other then specified in the task requirements btw